docs/blog/posts/announcing-keptn-v2-release-candidate.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2024-03-07
+authors: [agardnerit]
+description: >
+  Announcing the Official Release Candidate for Keptn v2
+categories:
+  - Installation
+  - Upgrade
+comments: true
+---
+
+# Announcing the Official Release Candidate for Keptn v2
+
+The Keptn project is proud to announce a release candidate for what will become Keptn v2.
+
+This release candidate will provide the project with enough end-user validation and bug-finding so that we can release Keptn v2.
```

Comment:
```suggestion
This release candidate provides the project
with enough end-user validation and bug-finding so that we can release Keptn v2.
```
Flo, maybe you can do better on the wording?